### ISSUE TITLE
PDF reader: RTL text copying/selection/highlighting is handled as LTR text and gets reversed

### ISSUE THREAD

**Issue by mrtcode** [zotero/zotero#2509]
https://forums.zotero.org/discussion/96022/arabic-script-and-searchable-pdfs-in-beta-6-0-5
https://forums.zotero.org/discussion/98567/rtl-language-highlights-not-working-correctly

**Comment by mrtcode** [comment]
Although this doesn't work well on PDF.js either (unpredictable selection, although text is not reversed), therefore fixing this might be challenging.

**Comment by helsingi** [comment]
Hi, I am working with a lot of Arabic text and notice regularly that the pdf reader in zotero recognizes the OCRed arabic text in reverse, i.e., LTR instead of RTL. Is there anything I could do or are you working on a fix? Thanks.

**Comment by mrtcode** [comment]
@helsingi Could you upload example PDF files here? Single page per PDF file is enough.

**Comment by helsingi** [comment]
> @helsingi Could you upload example PDF files here? Single page per PDF file is enough.

[abbas 2016 (copy).pdf](https://github.com/zotero/zotero/files/14238614/abbas.2016.copy.pdf)
[ghunaym 2013 (copy).pdf](https://github.com/zotero/zotero/files/14238615/ghunaym.2013.copy.pdf)
[hussein 1973 (copy).pdf](https://github.com/zotero/zotero/files/14238616/hussein.1973.copy.pdf)
[sharaf 1992 (copy).pdf](https://github.com/zotero/zotero/files/14238617/sharaf.1992.copy.pdf)

**Comment by mrtcode** [comment]
@helsingi, thank you for the examples. However, it seems they work better on Zotero 7 beta, doesn't it?

**Comment by helsingi** [comment]
> @helsingi, thank you for the examples. However, it seems they work better on Zotero 7 beta, doesn't it?

Not really. Just installed zotero 7 on windows and tried searching (ctrl+f) within a pdf attachment, unfortunately the same issue persists.
![Capture1](https://github.com/zotero/zotero/assets/129259619/c1375f28-8ee6-47f7-8009-6c7936a9186f)
In the screenshot you can see that the word I used is the reverse (LTR) of the actual word. When I search for the correct word no results show.

**Comment by mrtcode** [comment]
@helsingi Ok, at least with those PDF files it actually doesn't work well. Reopening the issue.
